src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -214,6 +214,19 @@ def __str__(self):
         return pformat(o)
 
 
+AZURE_DF_CLOUD = Cloud(
+    'AzureDFCloud',
+    endpoints=CloudEndpoints(
+        management='https://management-preview.core.windows-int.net/',
+        resource_manager='https://api-dogfood.resources.windows-int.net/',
+        gallery='https://df.gallery.azure-test.net/',
+        active_directory='https://login.windows-ppe.net',
+        active_directory_graph_resource_id='https://graph.ppe.windows.net/',
+        active_directory_resource_id='https://management.core.windows.net/',
+        microsoft_graph_resource_id='https://graph.ppe.windows.net/'),
+    suffixes=CloudSuffixes(
+        keyvault_dns='.vault-int.azure-int.net'))
```

Comment:
I prefer not to hard-code our testing environment in the public CLI code. We can [register a new cloud](https://docs.microsoft.com/en-us/cli/azure/manage-clouds-azure-cli?toc=%2Fcli%2Fazure%2Ftoc.json&bc=%2Fcli%2Fazure%2Fbreadcrumb%2Ftoc.json&view=azure-cli-latest#register-a-new-cloud) instead.

Reply:
> I prefer not to hard-code our testing environment in the public CLI code. We can [register a new cloud](https://docs.microsoft.com/en-us/cli/azure/manage-clouds-azure-cli?toc=%2Fcli%2Fazure%2Ftoc.json&bc=%2Fcli%2Fazure%2Fbreadcrumb%2Ftoc.json&view=azure-cli-latest#register-a-new-cloud) instead.

Good idea, I'll deprecate this part.